### PR TITLE
[MRG] MAINT add badge for readthedocs

### PR DESCRIPTION
**Pull request by glemaitre**
It might be good to have the badge to read the docs to check if the builds are not failing.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -16,10 +16,13 @@
...
 
-
+.. |ReadTheDocs| image:: https://readthedocs.org/projects/joblib/badge/?version=latest
+   :target: http://joblib.readthedocs.io/en/latest/?badge=latest
+   :alt: Documentation Status
+         
 The homepage of joblib with user documentation is located on:
 
-https://pythonhosted.org/joblib/
+https://joblib.readthedocs.io
```
Comment (aabadie) [inline README.rst]: so this means that readthedocs is the official Joblib documentation ?

**Comment by lesteve** [comment]
I am fine with merging this but just a comment: I would not go overboard with readthedocs, it is quite likely that we will move away from it at one point in the not so far future.

One reason is that to use the binder functionality we need to have the generated notebooks somewhere on github. IIRC for sphinx-gallery this is why we are moving away from readthedocs to and deploy the doc on gh-pages through CircleCI.

**Comment by GaelVaroquaux** [comment]
+1 to the comment of @lesteve

**Comment by glemaitre** [comment]
Ok that make sense

**Pull request closed without merging**